internal/apis/kfd/v1alpha2/eks/create/kubernetes.go

```diff
@@ -23,24 +24,34 @@ import (
 	"github.com/sighupio/furyctl/configs"
 	"github.com/sighupio/furyctl/internal/cluster"
 	"github.com/sighupio/furyctl/internal/template"
+	"github.com/sighupio/furyctl/internal/tool/awscli"
 	"github.com/sighupio/furyctl/internal/tool/terraform"
 	bytesx "github.com/sighupio/furyctl/internal/x/bytes"
 	execx "github.com/sighupio/furyctl/internal/x/exec"
 	iox "github.com/sighupio/furyctl/internal/x/io"
 	kubex "github.com/sighupio/furyctl/internal/x/kube"
+	netx "github.com/sighupio/furyctl/internal/x/net"
 )
 
 var (
-	errKubeconfigFromLogs = errors.New("can't get kubeconfig from logs")
-	errPvtSubnetNotFound  = errors.New("private_subnets not found in infra output")
-	errPvtSubnetFromOut   = errors.New("cannot read private_subnets from infrastructure's output.json")
-	errVpcCIDRFromOut     = errors.New("cannot read vpc_cidr_block from infrastructure's output.json")
-	errVpcCIDRNotFound    = errors.New("vpc_cidr_block not found in infra output")
-	errVpcIDNotFound      = errors.New("vpc id not found: you forgot to specify one or the infrastructure phase failed")
+	errKubeconfigFromLogs  = errors.New("can't get kubeconfig from logs")
+	errPvtSubnetNotFound   = errors.New("private_subnets not found in infra output")
+	errPvtSubnetFromOut    = errors.New("cannot read private_subnets from infrastructure's output.json")
+	errVpcCIDRFromOut      = errors.New("cannot read vpc_cidr_block from infrastructure's output.json")
+	errVpcCIDRNotFound     = errors.New("vpc_cidr_block not found in infra output")
+	errVpcIDNotFound       = errors.New("vpc id not found: you forgot to specify one or the infrastructure phase failed")
```

Comment:
let's give a better hint to the user:
```suggestion
	errVpcIDNotFound       = errors.New("vpcId not found: you forgot to specify one in the configuration file or the infrastructure phase failed")
```